### ISSUE TITLE
Add telemetry to protocol specification

### ISSUE THREAD

**Issue by cryptocode** [nanocurrency/protocol#9]
https://github.com/nanocurrency/nano-node/pull/2446

**Comment by wezrule** [comment]
telemetry_req
No payload

telemetry_ack
`header.extensions` contains the size of the telemetry data payload.

```
64 block_count
64 cemented_count
64 unchecked_count
64 account_count
64 bandwidth_cap
64 uptime
32 peer_count
8 protocol_version
256 genesis_block
8 major_version
8 minor_version
8 patch_version
8 pre_release_version
8 maker
64 timestamp
```

**Comment by Inkeliz** [comment]
I think it should specify the type, not jus the size. 

For comparation:

https://github.com/nanocurrency/protocol/blob/047eb69555d758aa870df4bc00848ab3be98378b/reference/protocol.ksy#L288-L296

https://github.com/nanocurrency/protocol/blob/047eb69555d758aa870df4bc00848ab3be98378b/reference/protocol.ksy#L352-L380


I think `blockcount`, for instance, is an uint64le, but it's not specified. I think it should be like the `vote_common`, with `type: u8le`.

**Comment by wezrule** [comment]
@Inkeliz 

Makes sense, thanks. https://github.com/nanocurrency/protocol/pull/12/files

**Comment by zhyatt** [comment]
Fixed in #12
